cargo-dist/src/tasks.rs

```diff
@@ -727,7 +743,7 @@ impl<'pkg_graph> DistGraphBuilder<'pkg_graph> {
 
         Ok(Self {
             inner: DistGraph {
-                is_init: dist_profile.is_some(),
+                is_init: desired_cargo_dist_version.is_some(),
```

Comment:
I changed this since `dist_profile` is no longer a decent "is this initted" proxy.